### DISCUSSION TITLE
Colour icons

### DISCUSSION THREAD

**Discussion by daniel-wojahn** [FlorianWoelki/obsidian-icon-folder#117] [Q&A] [answered]
Hi Florian,

great stuff! However, I am fairly new to the Obsidian universe and was wondering whether there is a way to assign a colour to these icons, too? I installed this CSS snippet (https://forum.obsidian.md/t/iterative-rainbow-folder-colors-css/21066) but it removes the icon and leaves me with a blank coloured square instead.

Happy for any recommendation on how to colour icons enabled via your wonderful plugin :)

**Comment by veitch123** [comment]
Yeah. I was thinking about this too. Perhaps allowing some desaturation or a colour caste. I just added my own SVGs and they don't fit at all with the theme. I'm wondering whether it would be possible to, not entirely change the colour (because the app icons would just be blank without a recognisable shape) but allow them to have a colour caste on them.

**Comment by veitch123** [comment]
There's an enhancement at https://github.com/FlorianWoelki/obsidian-icon-folder/issues/124 that might help

**Comment by FlorianWoelki** [comment]
Thank you for your discussion.

#124 is a current enhancement that will be developed soon. So stay tuned.

In the mean time, there is a general color adjustment that changes all the icons.

**Comment by FlorianWoelki** [comment, marked as answer]
Btw. this has been released some time ago :D
